cloudinary-test-common/src/main/java/com/cloudinary/test/AbstractUploaderTest.java

```diff
@@ -591,6 +594,13 @@ public void testCreateArchiveRaw() throws Exception {
 
     }
 
+    @Test
+    public void testCreateZipMultipleResourceTypes() throws Exception {
+        Map result = cloudinary.uploader().createZip(ObjectUtils.asMap("fully_qualified_public_ids",(new String[]{SRC_FULLY_QUALIFIED_IMAGE,SRC_FULLY_QUALIFIED_VIDEO})));
+        assertEquals(2, result.get("file_count"));
+        cloudinary.api().deleteResources(Arrays.asList(result.get("public_id").toString()), asMap("resource_type", "raw"));
```

Comment:
Even though it works without it, we should set `resource_type` to `auto` to demonstrate the 'correct' usage of the feature (in the may future it may fail without setting it).